numba/cuda/cudaimpl.py

```diff
@@ -56,6 +64,25 @@ def ptx_gridsize2d(context, builder, sig, args):
     return cgutils.pack_array(builder, [r1, r2])
 
 
+@register
+@implement('ptx.gridsize.3d', types.intp)
+def ptx_gridsize2d(context, builder, sig, args):
```

Comment:
Although it's not called by name (IIUC), this should probably be renamed to `ptx_gridsize3d`.